api.swagger.yaml

```diff
@@ -11807,7 +11817,49 @@ components:
         password:
           description: The password that the user logs in with
           type: string
-    layout-schema:
+    default-layout-schema:
+      type: object
+      properties:
+        title:
+          type: string
+        icon:
+          type: string
+        category:
+          type: string
+        theme:
+          type: string
+        module:
+          type: string
+        name:
+          type: string
+        path:
+          type: string
+        file:
+          type: string
+        plugin module:
+          type: string
+        plugin type:
+          type: string
+        description:
+          type: string
+        regions:
```

Comment:
This should be an array of objects like so:
{
    region: 'sidebar',
    region_label: 'Region 1',
}